controllers/api/v1alpha1/cfserviceinstance_types.go

```diff
@@ -25,8 +25,9 @@ import (
 )
 
 const (
-	UserProvidedType = "user-provided"
-	ManagedType      = "managed"
+	UserProvidedType     = "user-provided"
+	ManagedType          = "managed"
+	CredentialsSecretKey = "credentials"
```

Comment:
We already have a [constant](https://github.com/cloudfoundry/korifi/blob/89d5234409cc06abb6462b21cd34cf6f2435212e/tools/secrets.go#L10) for that key, no need of a new one. But if you use the utilities from the `tools/secrets.go` file, you should probably not need to reference it.